src/script/components/ServiceList.tsx

```diff
@@ -48,15 +48,30 @@ const ServiceList: React.FC<ServiceListProps> = ({
   noUnderline,
   services,
 }) => {
+  const handleKeyPress = (event: React.KeyboardEvent<HTMLDivElement>, service: ServiceEntity) => {
+    if (event.key === 'Enter' || event.key === ' ') {
```

Comment:
for those ones you can use `Util/KeyboardUtil` key comparison or even whole functions like `isSpaceKey`, `isEnterKey`

Reply:
does this apply to React's `event.key` property too?

Reply:
I've fixed it by using `KEY` constants instead of "Enter" and "  " strings